be/src/io/fs/buffered_reader.cpp

```diff
@@ -67,8 +68,7 @@ Status BufferedFileStreamReader::read_bytes(const uint8_t** buf, uint64_t offset
     while (has_read < to_read) {
         size_t loop_read = 0;
         Slice resutl(_buf.get() + buf_remaining + has_read, to_read - has_read);
-        IOContext io_context;
-        RETURN_IF_ERROR(_file->read_at(_buf_end_offset + has_read, resutl, io_context, &loop_read));
+        RETURN_IF_ERROR(_file->read_at(_buf_end_offset + has_read, resutl, &loop_read));
```

Comment:
Why not pass IOContext to function read_at()?  I understand that we use func BufferedFileStreamReader::read_ bytes() to read data, may be we can save IOContext to BufferedFileStreamReader when calling ScalarColumnReader:: init method?

Reply:
Here I just removed unused `IOContext`, not modify the origin logic.
Maybe we can add `IOContext` in `ScalarColumnReader` in next PR.